lesson1.tex

```diff
@@ -47,74 +46,57 @@
 
-    Try changing the Power setting (blue box on the diagram) on the block.
-    The Power setting is what determines how fast your robot moves. Ask the students the following questions:
-    \begin{itemize}
-        \item What happens if you set it to 100?
-        \item What happens if you set it to -100?
-        \item What happens if you set it to 0?
-        \item Set the power on your block to whatever speed you want.
-    \end{itemize}
-    \par
-    Let’s try to change the steering setting (red box on the diagram) on the block
-    The Steering determines which direction your robot will move.
-    \begin{itemize}
-        \item What happens if you set it to 100?
-        \item What happens if you set it to -100?
-        \item What happens if you set it to 0?
-    \end{itemize}
-    \par
-    Figure out how to make the robot turn to the right and then move forward
-    \par
-    Take a few minutes to play around with some of the other settings
-    \begin{itemize}
-        \item What happens if you change the Brake at End setting (orange box on the diagram)?
-        \item What happens if you change the Rotations to 2 or 3 (green box on the diagram)?
-        \item Can you figure out how to make the robot move forward for exactly 5 seconds (hint: there’s one setting we haven’t asked you to touch yet)?
-        \item  How would you make the robot move backwards and to the left at the same time using exactly one block?
-    \end{itemize}
+	\par Connect the LED block to the end of the display block. Show students that they are able to conenct multiple blocks together to have the robot do multiple things at once. The program should now look like this: 
+	
+	\procedureImage[0.5]{1-5.png}
+
+	\par Download the program, and show students the result. Again, have students replicate this on their own robots. 
+	
+	\par Finally, have students use the wait block. Experiment with the placement of the wait block as well as the duration of the wait block. Students should recognize that the program is read sequentially from the start block. 
+
+	\procedureImage[0.5]{1-6.png}
 
     \subsection{Extensions}
-      \begin{itemize}
-        \item Make the robot move forward a couple of feet.
-        \item Make the robot turn around completely.
-        \item Make the robot move forward again so that it returns to its starting location.
-        \item (advanced) Celebrate by having the robot make a sound of your choice.
-        \item (advanced) Make your robot do a figure-8 and stop in the same place it started.
-        \item (advanced) Make your robot try to spell out the first letter in one (or more) of your group members’ names. If you have time, try to spell out a full name!
-      \end{itemize}
+	\par Encourage students to look through the MINDSTORM programming interface and take a look at different blocks. Explain that in every session, you will be looking at most of these blocks in more detail. 
```

Comment:
Could you think of an extension that allows us to help students who finished early with another sort of assignment?